config/components/manager/controller_manager_config.yaml

```diff
@@ -13,10 +13,11 @@ leaderElection:
 controller:
   groupKindConcurrency:
     Job.batch: 5
+    Pod.: 5
```

Comment:
What is the default if not specified? Also, why only batch.Job and Pod frameworks are picked to have a custom value (5)?

Reply:
The default is one.

We could add all integrations, but this is just a default configuration that should work for most users.

Reply:
Would that make sense it defaults according to the number of CPUs available / cgroups CPU quota?

Reply:
This number is a default, users will have to pick up what makes sense for them.

We just have to pick some numbers that work at a reasonable scale (~50 nodes).
I picked the number based on the default number of workers in kcm.